packages/react/src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -459,11 +465,16 @@ const DashboardEditor = ({
    */
   const addCard = useCallback(
     (type) => {
-      // notify consumers that the card has been added if they're listening (they might want to tweak the card defaults)
-      const cardConfig = onCardChange
-        ? onCardChange(getDefaultCard(type, mergedI18n), dashboardJson)
+      const defaultCard = customGetDefaultCard // Use the default card specified by the consumer if it exists
+        ? customGetDefaultCard(type)
         : getDefaultCard(type, mergedI18n);
 
+      // notify consumers that the card has been added if they're listening (they might want to tweak the card defaults here)
+      const cardConfig =
+        onCardChange && !customGetDefaultCard
```

Comment:
is this right?  Should we call onCardChange even if they have a customGetDefaultCard or is the comment above the line incorrect?  Seems like they disagree.

Reply:
Right, the comment is left over and incorrect. To me it makes sense to call onCardChange when adding a new card IF there is no getDefaultCard passed, but if there is one passed, I would think the consumer would want to handle it there instead. I'll remove the misleading comment.

Reply:
Can we fix the comment?